lib/core/configurator.c

```diff
@@ -872,6 +872,26 @@ static int on_config_error_log_emit_request_errors(h2o_configurator_command_t *c
     return 0;
 }
 
+static int on_config_forward_informational(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, yoml_t *node)
+{
+    int value;
```

Comment:
Can't the type of the value be the enum that is being typedef'ed? I'd assume that it can be, and that you could then omit the cast when assigning the value to globalconf.